### PR TITLE
add Stdlib::Fqdn and Stdlib::Host

### PR DESCRIPTION
**Pull request by b4ldr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -413,6 +413,58 @@ Unacceptable input example:
...
+
+Unacceptable input example:
+
+```shell
+'www www.example.com'
+
+2001:DB8::1
+```
+
+#### `Stdlib::host`
```
Comment (ripienaar) [inline README.md]: `Stdlib::Host`

**Comment by b4ldr** [comment]
@ripienaar  thanks corrected

**Comment by pegasd** [comment]
If my PR #856 gets through, maybe these should also be updated with the new testing technique?

**Comment by pegasd** [comment]
@b4ldr Time to get cranking on those tests (-;

**Comment by b4ldr** [comment]
all updated assuming there are no more rubocop changes

**Pull request merged**